exporters/jaeger/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 include_directories(thrift-gen)
 
 find_package(Thrift REQUIRED)
+# vcpkg config recipe points to THRIFT_INCLUDE_DIR=...\thrift Ensure that the
```

Comment:
nit: missing period before `Ensure`?

Reply:
This was on two separate lines, but auto-formatter (`./ci/do_ci.sh format`) broke it.. I can add a period in between.